### ISSUE TITLE
Odd number of audio channels are never valid

### ISSUE THREAD

**Issue by matmat** [Ymagis/ClairMeta#54]
I believe that an odd number of audio channels are *never* valid. This is because physical AES pairs are used in almost all media block implementations. I have personal experience with Dolby Cat862/DSS200 fails at playback with odd number of channels.

Also most people find it unsafe with an even number of channels <6, even if I have no evidence of it NOT working. Especially I have successfully played many played many two-channel DCP:s. The usual thing done is to pad any unused channels (no. 1-6) with silence.

Reference 1:
http://isdcf.com/papers/ISDCF-Doc4-Audio-channel-recommendations.pdf

> Note 1. Not all channels need to be present in a given DCP. For instance, only the first 8 channels should be used when delivering 5.1 + HI/VI content. In all cases, an even number of channels shall be used.

Reference 2:
```
To: Main ISDCF <isdcf@lists.isdcf.com>
Date: Mon, 27 Jan 2014 20:18:29 +0000
Message-ID: <CF0BFC61.21F4F%wade.hanniball@nbcuni.com>
Subject: Re: [ISDCF] Channel count in Interop compositions
````
> In the past, Universal has made Interop DCPs of classic titles with ch. 1 & 2 only (L/R), and with ch. 1, 2, 3 & 4 only (L/R/C/LFE).  We've not experienced any playback problems in the field.  However, having thought about it more and just to be safe, we now include six channels of audio on all DCPs, putting null audio on "empty" channels.  And, yes, audio needs to be in AES pairs (no odd number of channels).

https://github.com/Ymagis/ClairMeta/blob/049f459544e9ec08da00d525d346704decd38458/clairmeta/settings.py#L113-L127

**Comment by remia** [comment]
Thanks for the details @matmat.

A first step would be to add a warning if an odd number of channels is detected, right ? Not sure what we could do next but I think we are not enough strict on sound.

**Comment by remia** [comment]
Implemented the basic check in #70

**Comment by remia** [comment]
#70 is now merged.

Please re-open if there still exists missing audio checks.
